eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs

```diff
@@ -441,3 +454,27 @@ instance
   where
   wrapFailed = ShelleyInAlonzoUtxowPredFailure . UtxoFailure
   wrapEvent = WrappedShelleyEraEvent . UtxoEvent
+
+mkScriptIntegrity ::
```

Comment:
It is better to place this smart constructor right next to the type itself and note this addition in the changelog